.github/actions/process-changelog/action.yml

```diff
@@ -41,7 +41,7 @@ runs:
           echo "Amending the changelog entries." >> $GITHUB_STEP_SUMMARY
         fi
           
-        ~/.composer/vendor/bin/changelogger write --use-version="$RELEASE_VERSION" --release-date="$RELEASE_DATE" $CHANGELOG_FLAG --no-interaction
+        ~/.composer/vendor/bin/changelogger write --use-version="$RELEASE_VERSION" --release-date="$RELEASE_DATE" $CHANGELOG_FLAG --no-interaction --yes
```

Comment:
Is `--no-interaction --yes` similar to not providing the interaction flag at all? This seems a little weird otherwise. 🤔

Reply:
I don't think so, if I read the Jetpack code correctly:

> ->addOption( 'yes', null, InputOption::VALUE_NONE, 'Default all questions to "yes" instead of "no". Particularly useful for non-interactive mode' )

https://github.com/Automattic/jetpack-changelogger/blob/f172de12a843254ef9e40433354969d00eeaea19/src/WriteCommand.php#L67

> Various edge cases will interactively prompt for information if possible. Use <info>--no-interaction</info> to avoid
this, along with <info>--yes</info> if you want to proceed through all prompts instead of stopping.

https://github.com/Automattic/jetpack-changelogger/blob/f172de12a843254ef9e40433354969d00eeaea19/src/WriteCommand.php#L83-L84